src/components/People.js

```diff
@@ -0,0 +1,71 @@
+import "../App.css";
+
+import React, { Component } from 'react'
+
+class People extends Component {
+    constructor(){
+        super()
+        this.state={
+            characterData:[],
+            userInput: ""
+        }
+
+    }
+
+    fetchCharacters=()=>{
+        fetch("https://ghibliapi.herokuapp.com/people")
+        .then(res=> res.json())
+        .then((data)=>{
+            this.setState({
+                characterData: data
+            })
+        })
+    }
+
+    componentDidMount(){
+        this.fetchCharacters()
+    }
+
+    handleUserInput=(event)=>{
+        this.setState({
+            userInput: event.target.value
+        })
+    }
+    handleSubmit=(event)=>{
+        event.preventDefault()
+     
+    }
+
+
+    render() {
+            let selectedCharactersObj = this.state.characterData.find((character)=>{
+                return character.name.toLowerCase() === this.state.userInput.toLowerCase() 
+            })
+
+        return (
+            <div className="people">
+                <h1>Search for a Person</h1>
+                <form onSubmit={this.handleSubmit}>
+                    <input 
+                    placeholder="Find Your Person" 
+                    type="text" 
+                    onInput={this.handleUserInput}
+                    value={this.state.userInput} 
+                    />
+                    <button>Submit</button>
+                </form>
+                <div className="character-container">
+                    <div>{selectedCharactersObj? selectedCharactersObj.name : "Not Found"}</div>
+                    <div>{selectedCharactersObj?.age}</div>
+                    <div>{selectedCharactersObj?.gender}</div>
+
+                </div>
```

Comment:
```suggestion
         { this.state.selectedCharacter ? characterInfo : null }
```
if `this.state.selectedCharacter` is **truthy**, that means that a search has been made, so `characterInfo `should display. If `this.state.selectedCharacter` is **falsy**, that means a search has NOT been made yet, so the screen should be blank.
Remember, **`characterInfo`'s value is dependent on conditional logic from an if statement**. It will either say "Not Found" or be JSX with all of the selected character's info.

So this ternary operator consists of **nested conditional logic**. A single piece of conditional logic is not sufficient because **there are _three_ options, not just two.**